README.md

```diff
@@ -25,14 +24,14 @@ Please review the theory behind FLOSS [here](doc/theory.md).
 To try FLOSS right away, download a standalone executable file from the releases page:
 https://github.com/fireeye/flare-floss/releases
 
-For a detailed description of *installing* FLOSS, review the documention
+For a detailed description of *installing* FLOSS, review the documentation
  [here](doc/installation.md).
 
 Standalone nightly builds:
-  - Windows 64bit: [here](http://s3.amazonaws.com/build-artifacts.floss.flare.fireeye.com/appveyor/dist/floss64.exe) 
-  - Windows 32bit: [here](http://s3.amazonaws.com/build-artifacts.floss.flare.fireeye.com/appveyor/dist/floss32.exe)
-  - Linux: [here](https://s3.amazonaws.com/build-artifacts.floss.flare.fireeye.com/travis/linux/dist/floss)
-  - OSX: [here](https://s3.amazonaws.com/build-artifacts.floss.flare.fireeye.com/travis/osx/dist/floss)
+  - Windows 64bit: TODO
+  - Windows 32bit: TODO
+  - Linux: TODO
+  - OSX: TODO
```

Comment:
i think we could maybe just remove this in favor of small point releases

Reply:
removed